drivers/seu/Kconfig.seu_intel_socfpga

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SEU_INTEL_SOCFPGA
+	bool
+	default y
+	depends on DT_HAS_INTEL_SOCFPGA_SEU_ENABLED
+	help
+	  To enable Intel SoC FPGA specific SEU driver
+
+if SEU_INTEL_SOCFPGA
+config SEU_MAX_CLIENT
+	int "SEU maximum client count"
+	default 5
+	help
+	  To set SEU maximum client count
+
+config MAX_TIMEOUT_MSECS
+	int "Maximum timeout in milliseconds"
+	default 1000
+	help
+	  Set maximum SiP SVC client connection timeout in milliseconds
```

Comment:
Would be nice to have this be more verbose, i.e. let's describe the effects of changing this option. Some sort of information in the help string why 1s was chosen as the default value would also be nice to have.